app/src/main/java/com/dmonzonis/nookpendium/CollectionActivity.kt

```diff
@@ -1,37 +1,51 @@
 package com.dmonzonis.nookpendium
 
 import android.os.Bundle
-import android.support.design.widget.TabLayout
-import android.support.v7.app.AppCompatActivity
-import android.support.v7.widget.LinearLayoutManager
-import android.support.v7.widget.RecyclerView
+import android.view.MenuItem
+import android.widget.Toast
+import androidx.appcompat.app.ActionBarDrawerToggle
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.tabs.TabLayout
 import kotlinx.android.synthetic.main.activity_collection.*
 import java.io.InputStream
 
 class CollectionActivity : AppCompatActivity() {
     private lateinit var viewAdapter: RecordListAdapter
     private lateinit var viewManager: RecyclerView.LayoutManager
+    lateinit var drawerToggle: ActionBarDrawerToggle
+    private var selectedGame: Int = R.string.game_acnh
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_collection)
 
-        // Get which collection needs to be opened from the value passed by the main menu activity
-        // TODO: get the selected tab
-        val filename = getString(R.string.fish_file)
+        // Initialize the navigation drawer toggle
+        drawerToggle = ActionBarDrawerToggle(this, drawerLayout, R.string.open, R.string.close)
+        drawerLayout.addDrawerListener(drawerToggle)
+        drawerToggle.syncState()
+
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
 
-        // Read the corresponding XML with the data and fill the records with it
-        val inputStream: InputStream = assets.open(filename)
-        val parser = RecordXmlParser(this)
-        val recordList: List<Record> = parser.parse(inputStream)
+        navGamesView.setNavigationItemSelectedListener {
+            // FIXME: If another tab other than 0 is selected when changing game, the data
+            // for what normally would be tab 0 will be put into whichever tab is active
+            when (it.itemId) {
+                R.id.miGameAcnh -> changeGame(R.string.game_acnh)
+                R.id.miGameAcnl -> changeGame(R.string.game_acnl)
+            }
+            true
```

Comment:
?

Reply:
apparently lambda expressions have to return true or some shit like that, if you remove it it won't compile